tfparse/__init__.py

```diff
@@ -44,7 +44,13 @@ def load_from_path(
     ]
 
     ret = lib.Parse(
-        s, stop_on_hcl_error, debug, allow_downloads, num_var_paths, c_var_paths
+        path,
+        stop_on_hcl_error,
+        debug,
+        allow_downloads,
+        workspace,
+        num_var_paths,
+        c_var_paths,
```

Comment:
I find the formatting + variable rename combo to be as valuable as the new param to be honest :)